src/space.js

```diff
@@ -79,6 +81,11 @@ class Space {
     return thread
   }
 
+  joinChat () {
```

Comment:
Instead of having a `space.joinChat()` method we should use the existing `space.joinThread()` method. This would make the API the same for developers that are already familiar with threads.

To join a ghostChat you could simply call:
```js
const thread = await space.joinThread('name', { ghost: true })
```

Reply:
We should also add this to the jsdoc on `joinThread`.